timescaledb/how-to-guides/hyperfunctions/regression-functions.md

```diff
@@ -0,0 +1,13 @@
+# Regression functions
+Monotonectally whiteboard compelling mindshare before virtual action items.
+Dramatically deploy integrated "outside the box" thinking without premier
+experiences. Uniquely reconceptualize frictionless methods of empowerment
+without backend web services. Energistically e-enable ubiquitous data and
```

Comment:
Think this isn't meant to be here. ...

Reply:
This page isn't published, because there's no content yet. It's on the "after the 18th" list 😉